### ISSUE TITLE
Backend: Send e-mail 

### ISSUE THREAD

**Issue by TamaraHoock** [amosproj/amos2022ss04-digital-identity#9]
## User story
1. As a (HR employee)
2. I want / need users to receive mails (in different processes)
3. So that I am able to communicate with them (e.g. invitation to use DI)

## Acceptance criteria
* backend sends email to a given email-address

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Idontker** [comment]
Finished in sprint 5: 8b9cdf974b0592bb76a4d87a3d35fe9834c38272
